### ISSUE TITLE
Channel logo instead of addon logo in channel info

### ISSUE THREAD

**Issue by nikhilk0** [botallen/repository.botallen#125]
## Feature request
For Add-on Jiotv and Disney+ Hotstar
I'm submitting a proposal to make Kodi show channel logo instead of addon logo when changing channels or viewing channel info.

For reference

This is what jiotv does
![Screenshot (79)](https://user-images.githubusercontent.com/45728234/115896115-b551cd80-a478-11eb-9bcf-7be94782cbda.png)

This is what it should look like (taken from discovery+ addon)
![Screenshot (81)](https://user-images.githubusercontent.com/45728234/115896214-cf8bab80-a478-11eb-879b-30cc297254f3.png)

**Comment by prtkvashishth** [comment]
Hii

Which skin are you using?

**Comment by nikhilk0** [comment]
It's Arctic Zephyr 2

**Comment by Nzzyme** [comment]
+1 for this. I observed the same in Windows and Android. Is this being considered for a enhancement?

**Comment by nikhilk0** [comment]
I don't think this is implemented in the latest update. I'm still getting jiotv logo in some channels and nothing i.e. neither channel logo nor jiotv logo (in pvr, channel number is shown instead for such channels) for other channels. Hope you'll look into it further.

**Comment by nikhilk0** [comment]
Channel logos are now visible with the latest addon version.
Still get the jiotv logo for 'Extra HD channels', but for normal channels respective channel logos show up.

If you manage to get the channel logos instead of jiotv logo for extra channels, then it would be perfect. But in its current state this is very good and far better than just a blank space.
